From: David Sterba <dsterba@suse.cz>
Date: Fri, 7 Feb 2014 14:34:04 +0100
Patch-mainline: 3.14
Git-commit: 8051aa1a3d5aaa7bd4c062cad94d09c3d567ef2e
Subject: [PATCH] btrfs: reserve no transaction units in
 btrfs_ioctl_set_features

Added in patch "btrfs: add ioctls to query/change feature bits online"
modifications to superblock don't need to reserve metadata blocks when
starting a transaction.

Signed-off-by: David Sterba <dsterba@suse.cz>
Signed-off-by: Chris Mason <clm@fb.com>
---
 fs/btrfs/ioctl.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/fs/btrfs/ioctl.c
+++ b/fs/btrfs/ioctl.c
@@ -4652,7 +4652,7 @@ static int btrfs_ioctl_set_features(stru
 	if (ret)
 		return ret;
 
-	trans = btrfs_start_transaction(root, 1);
+	trans = btrfs_start_transaction(root, 0);
 	if (IS_ERR(trans))
 		return PTR_ERR(trans);
 
